src/pretty.nit

```diff
@@ -1269,10 +1291,11 @@ redef class AIfExpr
 				end
 			end
 
-			if has_else(v) then
-				while not v.current_token isa TKwelse do
-					v.consume v.current_token.text
-				end
+			v.consume_comments
+
+			# FIXME: for some unknown reasons, has_else can be true even if
+			# there is no `else` keyword but a `end` instead.
+			if has_else(v) and v.current_token isa TKwelse then
 
 				v.indent -= 1
```

Comment:
I do not see the weirdness: statements in a `if` are indented with a `\t`. Even the vim indenter agrees on this one.